### DISCUSSION TITLE
Time Manager Reports

### DISCUSSION THREAD

**Discussion by MelanieGr** [te-online/timemanager#160] [General]
Hallo, können nur Admins im Time Manager Reporte für alle User downloaden? Oder kann man die Rechte für die Reports auch einzeln vergeben? 
Danke

**Comment by te-online** [comment]
Hallo!

Da fehlt eine Übersetzung im Teil-Dialog, aber der beschreibt eigentlich alles:
![grafik](https://github.com/te-online/timemanager/assets/4127167/36b54fc6-485d-44a6-b34e-65c0d57a5049)

> Du kannst alle Zeineinträge sehen, während andere nur ihre eigenen Zeiteinträge sehen und ändern können.

Das gleiche gilt für Reports.

- Du kannst deine eigenen Zeiteinträge sehen
- Du kannst Zeiteinträge von anderen sehen, wenn diese in einem Kunden sind, den du mit anderen geteilt hast
